composer.json

```diff
@@ -101,8 +101,8 @@
         "api-platform/core": "^2.6",
         "doctrine/dbal": "^2.12 || ^3.0",
         "doctrine/orm": "^2.8",
-        "friendsofphp/php-cs-fixer": "^2.18",
-        "infection/infection": "^0.21",
+        "friendsofphp/php-cs-fixer": "^3.0.0",
+        "infection/infection": "^0.22.1",
```

Comment:
Could we use `^0.22` instead?

Reply:
No: as said, 0.22.1 is the only version allowing xdebug-handler v2: https://packagist.org/packages/infection/infection#0.22.0

Reply:
Oh, sorry, didn't read the first comment, looks good to me, thanks 🙂